### ISSUE TITLE
`darwinSystem` does not accept `system` argument

### ISSUE THREAD

**Issue by gytis-ivaskevicius** [LnL7/nix-darwin#319]
Hi, 
This particular issue is causing issues for my project, you can read more about it here https://github.com/gytis-ivaskevicius/flake-utils-plus/pull/6

Anyways, the fix I'd expect is for `darwinSystem` to accept the `system` argument, especially since M1 is becoming a thing (so sooner or later its going to be necessary anyways)

I understand that implementing M1 is quite a feat - but can we just have a `system` option stub till this gets implemented? I don't mind creating a PR for this.

**Comment by bphenriques** [comment]
Hi! Did you encounter a work-around for this?

**Comment by gytis-ivaskevicius** [comment]
Yeah, `hosts.<hostname>.builder = args: nix-darwin.lib.darwinSystem (removeAttrs args [ "system" ]);`
I think I got an idea how to fix it from fup side but have been quite busy the past few months

**Comment by montchr** [comment]
@gytis-ivaskevicius since #317 has been merged, is this still an issue?
